assets/js/atomic/blocks/product-elements/image/supports.ts

```diff
@@ -8,8 +8,9 @@ export const supports = {
 		__experimentalBorder: {
 			radius: true,
 		},
+		typography: {
+			fontSize: true,
+		},
```

Comment:
What is the function of typography on image block? I couldn't find it affecting anything.

Reply:
> What is the function of typography on image block? I couldn't find it affecting anything.

For the sale label (visible only when the product is in sale).

https://user-images.githubusercontent.com/4463174/155153605-b870790e-4b0e-4d22-83bb-a954e24abefe.mp4

Reply:
Ah! True, didn't have it visible on single product. 

Thinking if having control over the sale badge as a separate sale badge, or generic badge block wouldn't be better maybe. This way one could control how it looks (including border, typography, color etc) and apply across the board.

What do you think?

Reply:
> Thinking if having control over the sale badge as a separate sale badge, or generic badge block wouldn't be better maybe. This way one could control how it looks (including border, typography, color etc) and apply across the board.

I agree, but we need to create another block. I guess that it is outside of the scope of this project.